examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3641,6 +3642,17 @@ class GLTFParser {
 			const input = animationDef.parameters !== undefined ? animationDef.parameters[ sampler.input ] : sampler.input;
 			const output = animationDef.parameters !== undefined ? animationDef.parameters[ sampler.output ] : sampler.output;
 
+			if ( target.node === undefined ) {
+
+				if ( target.extensions && target.extensions[ 'KHR_animation_pointer' ] ) {
+
+					console.warn( 'THREE.GLTFLoader: KHR_animation_pointer is not yet supported.' );
+
+				}
```

Comment:
If the extension should be in `extensionsRequired`, this warning can be removed because warns here https://github.com/mrdoob/three.js/blob/42c59b4d516fd204f5f760e1f7daefbbbf18c49c/examples/jsm/loaders/GLTFLoader.js#L385-L389

Reply:
I don't think so, there's definitely the possibility of files that have optionally KHR_animation_pointer (so not required, but optional) that would still cause an exception here. Usually animation data in general is optional in files

Reply:
I don't think here is a good place to warn because it is in a loop therefore there is a chance that warning the same message bunch of times.

@donmccurdy Do you remember why we warn unknown extensions that are in `extensionsRequired` but don't do the ones that are only in `extensionsUsed`?

https://github.com/mrdoob/three.js/blob/1d98d60f8d2c2ed1f239965681ef6e494f38dd52/examples/jsm/loaders/GLTFLoader.js

Reply:
So are we aligned to

- remove the warning in line 3649 from this PR
- add warnings against `extensionsUsed` (as separate PR)?

Reply:
Yes, removing the warning from this PR sounds good to me. And making another PR for discussion whether to warn/error against extensionsUsed/Required is also good.

Reply:
Thanks – adjusted this one